google-cloud-logging/.autotest

```diff
@@ -1,15 +0,0 @@
-# -*- ruby -*-
```

Comment:
Why was this file deleted?

Reply:
It isn't included in the other gapic-only gems, and it seems less useful for gapic-only gems. What are your thoughts?

Reply:
I don't understand why this PR adding a new gem is removing files from an existing gem.

Reply:
Lol didn't notice the path. Adding it back.